docs/guides/misc/tor/dnssec.md

```diff
@@ -13,4 +13,4 @@ To re-run the above test, you also need to:
 
 #### Alternatives
 
-* An alternative would be using [DNSCrypt](https://github.com/pi-hole/pi-hole/wiki/DNSCrypt), but this leaves you in a position where you have to trust the [DNSCrypt resolver](https://www.dnscrypt.org/dnscrypt-resolvers.html) since your IP is not anonymized - [unless you configure DNSCrypt to route over Tor](https://github.com/DNSCrypt/dnscrypt-proxy/issues/399#issuecomment-214329222).
+* An alternative would be using [DNSCrypt (Archive)](https://web.archive.org/web/20210907204828/https://github.com/pi-hole/pi-hole/wiki/DNSCrypt) or [DNSCrypt-2.0 (Archive)](https://web.archive.org/web/20220617185156/https://github.com/pi-hole/pi-hole/wiki/DNSCrypt-2.0), but this leaves you in a position where you have to trust the [DNSCrypt resolver](https://www.dnscrypt.org/dnscrypt-resolvers.html) since your IP is not anonymized - [unless you configure DNSCrypt to route over Tor](https://github.com/DNSCrypt/dnscrypt-proxy/issues/399#issuecomment-214329222).
```

Comment:
The original wiki pages were removed because they were really outdated (last update was 2016). 
Pointing these links to web archive (outdated content) is a bad idea.

I see 3 solutions:

1. replace the old links with more current references to `DNSCrypt` (if there are good references);
2. completely rewrite **Alternatives** section (maybe using different solutions) and include up-to-date references;
3. completely remove the  **Alternatives** section.

Personally I think the last option would be better.